extensions/amp-bodymovin-animation/0.1/amp-bodymovin-animation.js

```diff
@@ -95,6 +126,52 @@ export class AmpBodymovinAnimation extends AMP.BaseElement {
     }
     return true;
   }
+
+  play_() {
+    if (this.playingState_ == PLAYING_STATE.PLAYING) {
+      return;
+    }
+
+    const message = JSON.stringify(dict({
+      'action': 'play',
+    }));
+    this.iframe_.contentWindow./*OK*/postMessage(message, '*');
```

Comment:
`this.iframe_` may not not exist when this is called. What we want to do is queue these requests until player is loaded and then send them down. So I recommend 
1- creating a `sendCommand_` method and using that here.
2- inside `sendCommand_`, chain to a `playerReadyPromise`, this requrest the 3p iframe to send a message back up when the SDK has loaded which would resolve this promise.

`amp-youtube` is a good example for this.